sqlx-core/Cargo.toml

```diff
@@ -158,6 +158,7 @@ memchr = { version = "2.4.1", default-features = false }
 num-bigint = { version = "0.4.0", default-features = false, optional = true, features = ["std"] }
 once_cell = "1.9.0"
 percent-encoding = "2.1.0"
+pin-project = "1.0.10"
```

Comment:
As in https://github.com/launchbadge/sqlx/pull/1860, this was already a transitive dependency :+1: